blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServerBuilder.scala

```diff
@@ -398,12 +404,11 @@ class BlazeServerBuilder[F[_]](
 
 object BlazeServerBuilder {
   @deprecated("Use BlazeServerBuilder.apply with explicit executionContext instead", "0.20.22")
-  def apply[F[_]](implicit F: ConcurrentEffect[F], timer: Timer[F]): BlazeServerBuilder[F] =
-    apply(ExecutionContext.global)
+  def apply[F[_]](D: Dispatcher[F])(implicit F: Async[F]): BlazeServerBuilder[F] =
```

Comment:
Let's remove this (now modified) overload as well.  It's been suggested we should just default the `ExecutionContext` to global, which I might do in a separate PR.